autorally_core/src/StateEstimator/BlockingQueue.h

```diff
@@ -90,6 +90,18 @@ class BlockingQueue {
     return boost::optional<WorkType>(element);
   }
 
+  // TODO this might not make sense - will assess later
```

Comment:
Why did you add this?  It doesn't seem to be used, if not lets remove it from the PR